src/main/app/forms/validation/validators/fractions.ts

```diff
@@ -24,7 +24,7 @@ export class FractionsConstraint implements ValidatorConstraintInterface {
     if (value == null) {
       return true
     }
-    const regex = '^\\d+\\.*\\d{' + min + ',' + max + '}\$'
+    const regex = '^[-]?\\d+\\.*\\d{' + min + ',' + max + '}\$'
```

Comment:
do we have field validation that relied on the fact that this didn't accept negatives?

Reply:
I don't know, but I believe we should then add `@Min(0, {...})`

Reply:
No test is broken when I updated it. Models are so easy to unit tests that this kind of cases should be covered.

Reply:
I have checked it again. All usages of `@Fractions` in models have also has also `@Min` or `@Positive` annotation.

The only exception is `SelfEmployed` where we want to be able to provide negative value.